addons/l10n_fr_pos_cert/models/account_bank_statement.py

```diff
@@ -10,7 +10,8 @@ class AccountBankStatement(models.Model):
 
     def unlink(self):
         for statement in self.filtered(lambda s: s.company_id._is_accounting_unalterable() and s.journal_id.pos_payment_method_ids):
-            raise UserError(_('You cannot modify anything on a bank statement (name: %s) that was created by point of sale operations.') % (statement.name,))
+            if statement.line_ids or statement.difference:
+                raise UserError(_('You cannot modify anything on a bank statement (name: %s) that was created by point of sale operations.') % (statement.name,))
```

Comment:
you probably want `statement.state != 'open'`
Also, cleanup this loop to be more efficient:
```
for statement in self:
    if not statement.company_id._is_accounting_unalterable() or not statement.s.journal_id.pos_payment_method_ids:
        continue
    if statement.state != 'open':
        raise UserError(_("You cannot modify anything on a bank statement (name: %s) that was created by point of sale operations.") % statement.display_name)
```

Reply:
yes thx :)